src/pages/UserFeed.jsx

```diff
@@ -0,0 +1,49 @@
+import axios from 'axios'
+import { useEffect } from 'react'
+import { FeedPageWrapper, Post, PostsWrapper } from "components/Reusable"
+import { useAuth, usePosts } from 'contexts'
+import styles from 'components/Reusable/reusable.module.css'
+import { ACTION_INIT_USER_FEED } from 'utils'
+
+const FeedPosts = PostsWrapper(Post)
+
+const UserFeedSection = () => {
+    const { postsDispatch } = usePosts()
+    const { getUserToken } = useAuth()
+
+    // you may ignore this for now, this is a setup for the user feed
+    useEffect(() => {
+        (async () => {
+            const response1 = await axios.get('/api/users')
+            const mohit = response1.data.users.filter(user => user.username === 'coolmohit')
+            const peter = response1.data.users.filter(user => user.username === 'ranjanchauhan')
```

Comment:
suggestion - you can check the status code of the api call

Reply:
yes, this is only for now. ill remove this code once all the features are added.